helios-client/src/main/java/com/spotify/helios/common/descriptors/TaskStatus.java

```diff
@@ -111,17 +115,19 @@ public TaskStatus(@JsonProperty("job") final Job job,
     this.throttled = Optional.fromNullable(throttled).or(ThrottleState.NO);
     this.ports = Optional.fromNullable(ports).or(EMPTY_PORTS);
     this.env = Optional.fromNullable(env).or(Maps.<String, String>newHashMap());
+    this.containerError = Optional.fromNullable(containerError).or("");
   }
 
   public Builder asBuilder() {
     return newBuilder()
-        .setJob(job)
-        .setGoal(goal)
-        .setState(state)
-        .setContainerId(containerId)
-        .setThrottled(throttled)
-        .setPorts(ports)
-        .setEnv(env);
+            .setJob(job)
```

Comment:
I've configured IntelliJ to use the Google style guide and it *refuses* to let me have these at the indent level I found them.